### PR TITLE
Do not review, for testing only

### PR DESCRIPTION
**Pull request by VladimirReshetnikov**

### CODE REVIEW

**Comment by VladimirReshetnikov** [comment]
@dotnet-bot Test this please

**Comment by VladimirReshetnikov** [comment]
@Jenkins Test this please

**Comment by VladimirReshetnikov** [comment]
@Jenkins Test this please.

File: src/Compilers/Core/Portable/MetadataReader/MetadataDecoder.cs [hunk 1]
```diff
@@ -427,38 +419,37 @@ private TypeSymbol GetTypeOfTypeRef(TypeReferenceHandle typeRef, out bool isNoPi
...
             }
-            else if (tokenType == HandleKind.ModuleReference)
+
+            if (tokenType == HandleKind.ModuleReference)
             {
                 return LookupTopLevelTypeDefSymbol(Module.GetModuleRefNameOrThrow((ModuleReferenceHandle)tokenResolutionScope),
-                                                   ref fullName,
-                                                   out isNoPiaLocalType);
+                    ref fullName,
+                    out isNoPiaLocalType);
```
Comment (VladimirReshetnikov) [inline src/Compilers/Core/Portable/MetadataReader/MetadataDecoder.cs]: Revert this formatting change.

**Comment by jaredpar** [comment]
This has been open for 9 days now, can we close?

**Pull request closed without merging**